config/secrets.yml

```diff
@@ -60,19 +59,14 @@ development: *default
 
 test:
   <<: *default
-  db_adapter: <%= ENV["test_db_adapter"] %>
```

Comment:
since we use the same database container across environments, we no longer need the test_x environment variables. can just use the defaults from earlier in yml